Author: Steve Kemp <skx@debian.org>
Description: Include <time.h> to fix segfault on some architectures.
Bug-Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=315969
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/

--- a/msgsnarf.c
+++ b/msgsnarf.c
@@ -23,6 +23,7 @@
 #include <nids.h>
 #include <pcap.h>
 #include <pcaputil.h>
+#include <time.h>
 
 #include "buf.h"
 #include "decode.h"
--- a/sshow.c
+++ b/sshow.c
@@ -15,6 +15,7 @@
 
 #include <sys/types.h>
 #include <sys/times.h>
+#include <time.h>
 
 #include <netinet/in_systm.h>
 #include <netinet/in.h>
